gui/src/main/java/io/bisq/gui/main/MainView.java

```diff
@@ -27,7 +27,11 @@
 import io.bisq.core.app.BisqEnvironment;
 import io.bisq.core.exceptions.BisqException;
 import io.bisq.gui.Navigation;
-import io.bisq.gui.common.view.*;
+import io.bisq.gui.common.view.CachingViewLoader;
+import io.bisq.gui.common.view.FxmlView;
+import io.bisq.gui.common.view.InitializableView;
+import io.bisq.gui.common.view.View;
+import io.bisq.gui.common.view.ViewLoader;
```

Comment:
Please don't expand these imports or the others that have been expanded below. The built-in IDEA settings should handle this correctly when running `Optimize Imports`. You may not be sync'd up correctly with those settings, or perhaps you're using some other IDE, in which case you'll need to take greater care to get it right manually.

In any case, please keep PRs atomic and avoid mixing in unrelated changes.

Reply:
Sorry, I didn't see `.gitignore` ignores `.idea` but actually keeps code style.
I've optimized imports using Bisq code style.